board/opentrons/ot2/post-image.sh

```diff
@@ -9,17 +9,16 @@ GENIMAGE_TMP="${BUILD_DIR}/genimage.tmp"
 
 sed -i -e 's/console=ttyAMA0,115200//' "${BINARIES_DIR}/rpi-firmware/cmdline.txt"
 
+overlays_to_add="dtoverlay="
+
 for arg in "$@"
 do
 	case "${arg}" in
 		--add-pi3-miniuart-bt-overlay)
 		if ! grep -qE '^dtoverlay=' "${BINARIES_DIR}/rpi-firmware/config.txt"; then
 			echo "Adding 'dtoverlay=pi3-miniuart-bt' to config.txt (fixes ttyAMA0 serial console)."
-			cat << __EOF__ >> "${BINARIES_DIR}/rpi-firmware/config.txt"
+			overlays_to_add="${overlays_to_add}pi3-miniuart-bt"
```

Comment:
I'm really sorry, I was completely wrong about this - you do want several dtoverlay stanzas, this doesn't work - it parses the second overlay as a param for the first. Revert this commit and we can merge it